os/kernel/binary_manager/binary_manager_callback.c

```diff
@@ -189,7 +189,7 @@ int binary_manager_send_statecb_msg(int recv_binidx, char *bin_name, uint8_t sta
 	int ret;
 	int send_count;
 	mqd_t send_mq;
-	mqd_t recv_mq;
+	mqd_t recv_mq = NULL;
```

Comment:
I think you had better modify recv_mq logic itself without this modification.

Reply:
It is hard to modify recv_mq  logic.
So it has to initialize.